src/networking/payload/payload.rs

```diff
@@ -0,0 +1,17 @@
+use super::packet::Packet;
+
+
+
+
+
+pub trait Ipv8Payload {
+
+  // // fn serialize(&self) -> Vec<PackingIdentifier>;
```

Comment:
Is this supposed to be commended out twice

Reply:
nope